front/src/components/boxs/device-in-room/DeviceRow.jsx

```diff
@@ -24,7 +24,7 @@ const DeviceRow = ({ children, ...props }) => {
       />
     );
   }
-  if (props.deviceFeature.type === 'dimmer') {
+  if (['dimmer', 'brightness'].includes(props.deviceFeature.type)) {
```

Comment:
Can we extract this to a file const at top level?

Reply:
And use DEVICE_FEATURE_TYPES const instead of hard coded strings?

Reply:
Done with rebase